ContactList/ContactList/UpdateViewController.swift

```diff
@@ -98,24 +98,48 @@ class UpdateViewController: UIViewController {
         } else {
             runCooseImageHandler()
         }
+    }    
+}
+
+extension UpdateViewController: UIImagePickerControllerDelegate, UINavigationControllerDelegate{
+    
+    func imagePickerController(_ picker: UIImagePickerController, didFinishPickingMediaWithInfo info: [UIImagePickerController.InfoKey : Any]) {
+        let choosenImage = info[UIImagePickerController.InfoKey.originalImage] as! UIImage
+        changeCurrentImage(imageState: .changed(newImage: choosenImage))
+        dismiss(animated:true, completion: nil)
     }
     
-    func changeCurrentPersonCopy() {
+    func imagePickerControllerDidCancel(_ picker: UIImagePickerController) {
+        dismiss(animated: true, completion: nil)
+    }
+}
+
+extension UpdateViewController: UITextFieldDelegate {
+    
+    func textFieldShouldReturn(_ textField: UITextField) -> Bool {
+        textField.resignFirstResponder()
+        return false
+    }
+}
+
+private extension UpdateViewController {
+    
+    private func changeCurrentPersonCopy() {
```

Comment:
`private extension` makes all the included methods `private`, so methods modifiers may be removed.